features/cellular/framework/API/CellularDevice.h

```diff
@@ -96,6 +96,21 @@ class CellularDevice {
      */
     virtual CellularContext *create_context(FileHandle *fh = NULL, const char *apn = NULL) = 0;
 
+    /** Creates a new CellularContext interface. This API should be used if serial is UART and PPP mode used.
+     *  CellularContext created will use data carrier detect to be able to detect disconnection much faster in PPP mode.
+     *  UARTSerial usually is the same which was given for the CellularDevice.
+     *
+     *  @param serial       UARTSerial used in communication to modem. If null then the default file handle is used.
+     *  @param apn          access point to use with context, can be null.
+     *  @param dcd_pin      Pin used to set data carrier detect on/off for the given UART
+     *  @param active_high  a boolean set to true if DCD polarity is active low
+     *
+     *  @return         new instance of class CellularContext or NULL in case of failure
+     *
+     */
+    virtual CellularContext *create_context(UARTSerial *serial, const char *apn, PinName dcd_pin = NC,
```

Comment:
This and the other declaration should also still have the `= NULL` default on `apn` I assume?

Reply:
Yep, added.